observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -165,3 +181,154 @@ spec:
 ----
 
 . Verify that logs are visible in the Log section of the Observe tab in the OpenShift web console.
+
+=== Quick Start with OTEL
+
+To configure OTLP ingestion and enable the OTEL data model, follow these steps:
+
+:FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* Cluster administrator permissions
+
+.Procedure
+
+. Install the `OpenShift Logging` and `Loki` Operators from OperatorHub.
+
+. Create a `LokiStack` custom resource (CR) in the `openshift-logging` namespace:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2022-06-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
```

Comment:
This secret needs to be created as well and its contents depend on which object storage the customer is using.

Reply:
Added a Note for this.